### PR TITLE
Slack notification of release draft

### PR DESCRIPTION
**Pull request by aslepakurov**
## Description of changes in release / Impact of release:
Add notification to #services-satellite on draft release creation

### Is this a breaking change?
  - [ ] Yes
  - [X] No

### If you answered Yes then describe why is it so
(insert text here)

#### Additional details go here
Message example:
https://getverygood.slack.com/archives/C012K5W6TKJ/p1607353413018500

### CODE REVIEW

**Comment by aslepakurov** [comment]
Screenshot 2020-12-07 at 17 06 17

File: .circleci/slack_notify.sh [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+[[ $body = "" ]] &&
+  body_text="<empty>" ||
+  body_text=body
+
+[[ $is_draft = "true" ]] &&
+  pretext="New <$release_url|release draft> was created" ||
+  pretext="New <$release_url|release> was published"
+
+curl --data-urlencode \
+  "payload={\"attachments\": [{\"fallback\":\"$pretext\", \"pretext\":\"$pretext\", \"color\":\"#40E0D0\", \"fields\":[{\"title\":\"Version\", \"value\":\"$tag\"}, {\"title\":\"Author\", \"value\":\"<$author_url|$author>\"}, {\"title\":\"maxOS asset\", \"value\":\"<$mac_url|$mac_name>\"}, {\"title\":\"Linux asset\", \"value\":\"<$linux_url|$linux_name>\"}, {\"title\":\"Release body\", \"value\":\"$body_text\"}]}]}" \
```
Comment (vvfesik) [inline .circleci/slack_notify.sh:25]:
```suggestion
  "payload={\"attachments\": [{\"fallback\":\"$pretext\", \"pretext\":\"$pretext\", \"color\":\"#40E0D0\", \"fields\":[{\"title\":\"Version\", \"value\":\"$tag\"}, {\"title\":\"Author\", \"value\":\"<$author_url|$author>\"}, {\"title\":\"macOS asset\", \"value\":\"<$mac_url|$mac_name>\"}, {\"title\":\"Linux asset\", \"value\":\"<$linux_url|$linux_name>\"}, {\"title\":\"Release body\", \"value\":\"$body_text\"}]}]}" \
```

File: .circleci/config.yml [hunk 2]
```diff
@@ -188,7 +188,10 @@ jobs:
           at: ./artifacts
       - run:
           name: Publish Release
-          command: ghr -draft -t ${GITHUB_TOKEN} -u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -n ${CIRCLE_TAG} ${CIRCLE_TAG} ./artifacts
+          command: |
+            ghr -draft -t GITHUB_TOKEN-u ${CIRCLE_PROJECT_USERNAME} -r ${CIRCLE_PROJECT_REPONAME} -c ${CIRCLE_SHA1} -n ${CIRCLE_TAG} ${CIRCLE_TAG} ./artifacts
```
Comment (rozenbom) [inline .circleci/config.yml:192]: Looks like typo: `GITHUB_TOKEN-u` ?

File: .circleci/config.yml [hunk 3]
```diff
@@ -188,7 +188,10 @@ jobs:
...
+            ./.circleci/slack_notify.sh
```
Comment (rozenbom) [inline .circleci/config.yml:193]: In my current [PR](https://github.com/verygoodsecurity/vgs-satellite/pull/109) I've created a scripts dir in order to consolidate such scripts. Please consider creating scripts dir here and moving slack_notify.sh there.

File: .circleci/slack_notify.sh (hunk 1 above)
Comment (aslepakurov) [reply .circleci/slack_notify.sh]: 👍

File: .circleci/config.yml (hunk 2 above)
Comment (aslepakurov) [reply .circleci/config.yml]: 👍

File: .circleci/config.yml (hunk 3 above)
Comment (aslepakurov) [reply .circleci/config.yml]: 👍

**Review by rozenbom — APPROVED** [review]

**Pull request merged**